sigstore/_cli.py

```diff
@@ -221,6 +221,12 @@ def _add_shared_oidc_options(
         default=os.getenv("SIGSTORE_OIDC_ISSUER", DEFAULT_OAUTH_ISSUER_URL),
         help="The OpenID Connect issuer to use (conflicts with --staging)",
     )
+    group.add_argument(
+        "--oidc-disable-default-browser",
```

Comment:
I think this flag name might be a little ambiguous: what it's doing is forcing the OOB flow, which is conceptually untied to the "default" browser. It's also doing an OAuth2 flow, so I think we should emphasize that rather than OIDC (OIDC refers to the "shape" of the claims that come out of the flow).

What do you think about `--oauth-no-browser` or `--oauth-force-oob`?

Reply:
(I have a slight preference for the latter, since it'll be consistent with the environment suggestion below. But either is fine IMO; your call.)

Reply:
I renamed `--oauth-force-oob`, even though I think prepending `--oidc-xxx` wold be more consistent with other commands if it's under the oidc group. Lmk

Reply:
Yeah, it's a little inconsistent, but IMO it's worth it for the compatibility/symmetry with the existing flag. 

There might be a better section to put this under; I'll take a look.